### DISCUSSION TITLE
How to convert animated webp into apng

### DISCUSSION THREAD

**Discussion by cristalcho** [ImageMagick/ImageMagick#2764] [Help]
I am trying to convert animated webp into APNG. At the ChangeLog, I check a current Imagemagick support apng formats.
I tried 
`convert animated.webp apng:animated.png`

It did not work and I got this error
` convert: delegate failed ''ffmpeg' -nostdin -v -1 -i '%M%%d.pam' -plays %I '%u.%m' 2> '%u'' @ error/delegate.c/InvokeDelegate/1961. `

I installed ffmpeg and libpng with apng patch, could you give me the example how to convert into apng format?

**Comment by fmw42** [comment]
I believe this should work as follows, but I am getting an ffmpeg error also.

 `magick -delay 50 -loop 0 lena.jpg mandril3.jpg monet2.jpg zelda1.jpg APNG:anim.apng`

magick: delegate failed `'avconv' -nostdin -v -1 -i '%M%%d.pam' -plays %I '%u.%m' 2> '%u'' @ error/delegate.c/InvokeDelegate/1898.

I think the ImageMagick developers will need to look into this, since I tested this a couple releases ago and it worked.
